website/static/js/fileViewTreebeard.js

```diff
@@ -63,11 +63,13 @@ function FileViewTreebeard(data) {
             this.options.showTotal = Math.floor(containerHeight / this.options.rowHeight) + 1;
             this.redraw();
         },
-        lazyLoadOnLoad: function(tree) {
+        lazyLoadOnLoad: function(tree, event) {
```

Comment:
If it says undefined that is most likely because treebeard has not updated for you. You might need to run webpack or npm install againt. 

It is already in ontogglefolder, onlazyload gets it now as well. It has to be there because the scrolling function is inside lazyloadonload and it runs every time on lazyload which is clearly unwanted, there needs to be a state that checks if onlazyload should scroll or not, if it always scrolls every toggle will make this run. 

This change basically looks to see if lazyloadonload is loading as part of an event (click event) and not programmatically opening.

Let me know if it still doesn't work for you, it works for me locally, I might have missed a step in build or package commit number update but let's make sure.